.env.example

```diff
@@ -4,3 +4,5 @@ DATABASE_PW = database-pasword
 DATABASE_HOST = database-host
 DATABASE_PORT = database-port
 DATABASE_TEST = database-name-test
+MAILER_DEFAULT_URL_OPTIONS_HOST = MAILER_DEFAULT_URL_OPTIONS_HOST
+MAILER_DEFAULT_URL_OPTIONS_PORT = MAILER_DEFAULT_URL_OPTIONS_PORT
```

Comment:
add an extra line here

Reply:
Thank you for reviewing us, we will apply the changes immediately.

Reply:
@OswaldoPineda  Should i put this line before MAILER_DEFAULT_URL_OPTIONS_HOST  or after both of the new lines added?